integration/terraform/ec2/localstack/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "ec2_instance_type" {
+  type = string
+  default = "t3a.xlarge"
+}
+
+variable "key_name" {
+  type = string
+  default = "cwagent-integ-test-key"
+}
+
+variable "iam_instance_profile" {
+  type = string
+  default = "CloudWatchAgentServerRole"
+}
+
+variable "vpc_security_group_ids" {
+  type = list(string)
+  default = ["sg-013585129c1f92bf0"]
```

Comment:
Is there anything special or interesting about this security group?
allows incoming connections on port 443? 80?

Reply:
This is the security group that I have been using for testing. Mandatory open is 22 (ssh -ip must be open to all address as the connect comes from runner-), 4566 (secure and unsecure -can come within the same security group-) on in. Out, not sure since the default is allow all (I assume 80, 443, and 4556 out are the only required).